### ISSUE TITLE
espflash example

### ISSUE THREAD

**Issue by TyberiusPrime** [faern/esp-ota#3]
I couldn't get the espflash save-image example to work.

The command line is wrong / bitrotted, apparently. 
It' needs something like  '--chip esp32s3' instead of just "ESP32-S3", but then it complained that my app is too large for the default partition.

And when I passed -T for a partition table, it wanted me to pass in '--merge' as well, which gave a file that was 4x the size I expected (and exceeding my partition size). Then I tried -P as well, that gave a file about the right size, but esp-idf rejected it because of a hash mismatch.

The esptool example worked right away though :).

Just wanted to let you know,
and maybe it helps the next person going down this path.

**Comment by faern** [comment]
Ah, yes. The readme has the format that is correct for `espflash 1.7.0`. In espflash `2.1.0` the corresponding command is:
```
espflash save-image --chip esp32c3 target/release/my-app my-app.bin
```
And that works fine for me. Probably a good idea to update the readme! espflash 2 is the thing to use these days!

**Comment by faern** [comment]
Fixed in f0a5281.

I have no idea what the other problems you got were about. I don't think they are related to this crate at all. If it's still a problem for you, consider opening issues in the espflash repository maybe?
